lib/msal-core/src/telemetry/TelemetryEvent.ts

```diff
@@ -32,6 +31,24 @@ export default class TelemetryEvent {
     public stop(): void {
         // Set duration of event
         this.setElapsedTime(+Date.now() - +this.startTimestamp);
+
+        if ("performance" in window && window.performance.mark && window.performance.measure) {
```

Comment:
Can this boolean check be factored out since you do this check again?

Reply:
Yeah, thats a good call.

Reply:
Done.